README.md

```diff
@@ -726,7 +726,8 @@ Before you head over, read the [Contribution Guide](CONTRIBUTING.md) first. You
 
 - ### Refactoring
 
-  (In Progress)
+  - **Course**
```

Comment:
Change "Course" title to "Video" please :)

Reply:
Done. Thanks for your comment.